src/main/connector/system/Linux/Linux.yaml

```diff
@@ -143,34 +143,23 @@ monitors:
     simple:
       type: multiInstance
       sources:
-        physicalDiskInformation:
-          # id;reads;readsMerged;readTime;writes;writesMerged;writeTime;ioTime
+        physicalDiskActivity:
+        #ReadOperations;WriteOperations
           type: commandLine
           commandLine: cat /proc/diskstats
-          selectColumns: 3,4,5,7,8,9,11,13
           computes:
-          - type: divide
-            column: 2
-            value: 1000
-          - type: divide
-            column: 5
-            value: 1000
-          - type: divide
-            column: 8
-            value: 1000
+          - type: awk
+            script: |
+              $3 ~ /^[shv]d[a-z]|dm-[0-9]+$/ {
+                print $3 ";" $6*512 ";" $10*512
+              }
       mapping:
-        source: ${source::physicalDiskInformation}
+        source: ${source::physicalDiskActivity}
         attributes:
           id: $1
-          system.disk.device: $1
         metrics:
-          system.disk.operations{direction:read}: $2

```

Comment:
Why are we removing these metrics? They are not available? (`system.disk.operations`, etc.)